docs/markdown/Design-rationale.md

```diff
@@ -251,10 +251,10 @@ Get the code
 --
 
 The code for this experiment can be found at [the Meson
-repository](https://sourceforge.net/p/meson/code/). It should be noted
-that it is not a build system. It is only a proposal for one. It does
-not work reliably yet. You probably should not use it as the build
-system of your project.
+repository](https://github.com/mesonbuild/meson). It should be noted
```

Comment:
Maybe it would also make sense to link to the tree for an old commit in the vicinity of when this document was written?

Reply:
I think readers will understand if they read the hopes and ambitions in the final paragraph and then follow the link, they'll see the project is successful.